lib/mapper.js

```diff
@@ -93,3 +98,43 @@ exports.alias = function(alias){
     previous_id: alias.previousId()
   }
 }
+
+/**
+ * Map page.
+ *
+ * @param {Page} page
+ * @return {Object}
+ * @api private
+ */
+exports.page = function(page){
+  var evtName = "[Segment Page]";
+  if (page.properties().name || page.properties().url) {
+    evtName += " " + page.properties().name || page.properties().url;
+  }
+  return {
+    user_id: page.userId(),
+    properties: page.properties(),
+    event: evtName,
+    timestamp: page.timestamp().getTime() / 1000
+  };
+};
+
+/**
+ * Map screen.
+ *
+ * @param {Screen} screen
+ * @return {Object}
+ * @api private
+ */
+exports.screen = function(screen){
```

Comment:
This logic looks mostly the same as the `page` call, except for the name. It should be pulled into a function.